### ISSUE TITLE
Adding files to the script directory disables the extension

### ISSUE THREAD

**Issue by nickbudi** [p3lim/dotjs-universal#14]
Once I add any file inside the extension's folder, the extension is disabled in chrome with the message: This extension may have been corrupted. Repairing the extension removes any files I've added to /scripts and /styles.

I'm unable to disable the Extension Content Verification flag (chrome://flags/#extension-content-verification) in Chrome 40.0.2209.0 dev.

Any ideas?

**Comment by repton-infinity** [comment]
Getting the same thing. Currently running Chrome 39.0.2171.99 m on Win 8.1.

I tried adding a window.alert() to default.js. Got the popup first time I did anything, but then the extension was marked as corrupt and no more popups. Also Google told me it was disabled if I went to the store page.

**Comment by p3lim** [comment]
I can't think of anything that I could do to resolve this issue, the updates to Chrome has sadly killed this extension.

Until it is resolved (if ever), I suggest looking at the original extension or something like tampermonkey.

Feel free to submit patches or ideas of how to fix this.

**Comment by leesei** [comment]
It's sad that this extension stopped working.
I'm thinking of two workarounds:
1. Use Chrome's File System/HTML5 localStorage
https://developer.chrome.com/apps/app_storage
We have to *add UI* to allow user to copy and paste codes to put them into dotjs-universal's sandbox
It's better to embed an editor (AceEditor/CodeMirror) into the extension
As a plus, Chrome's File System can be sync'ed across browsers
2. ~~Use NPAPI~~
https://developer.chrome.com/extensions/npapi
But NPAPI is being phased out and WebStore would not accept extensions using
the new PepperAPI is also sandboxed so the first way is better

**Comment by geotheory** [comment]
RIP dotjs-universal

**Comment by judy** [comment]
Yeah, I'm really sad about this too. Still looking for other options, but nothing was quite as elegant. I'll go back to trying plain dotjs, but tampermonkey is a bit too heavy-handed for me.

**Comment by p3lim** [comment]
@leesei I had a plan on making something like you suggested (a full UI with editor), even with support for syncing the scripts with a GitHub repository, but I figured that Tampermonkey would be better in almost every case.
